paddle/fluid/operators/interpolate_op.cc

```diff
@@ -82,6 +82,18 @@ class InterpolateOpMaker : public framework::OpProtoAndCheckerMaker {
                          "bilinear interpolation and \"nearest\" for nearest "
                          "neighbor interpolation.")
         .SetDefault("bilinear");
+    AddAttr<bool>(
+        "align_corners",
+        "an optinal bool. Defaults to True. "
+        "If True, the centers of 4 corner pixels of the input and output "
+        "tensors are aligned, preserving the values at the corner pixels, "
+        "if Flase, are not aligned")
+        .SetDefault(true);
+    AddAttr<int>("align_mode",
+                 "(int, default \'0\'), align_corners mode , can be \'0\' "
+                 "for pytorch calculation method, can be \'1\' for "
+                 "tensorflow calculation method.")
```

Comment:
not to write pytorch and tf here, explain the index calucation by fomula

Reply:
fixed